jwst/associations/lib/rules_level2b.py

```diff
@@ -52,6 +54,84 @@
 # --------------------------------
 # Start of the User-level rules
 # --------------------------------
+@RegistryMarker.rule
+class Asn_Lv2Coron(
+        AsnMixin_Lv2Coron,
```

Comment:
`AsnMixing_Lv2Coron` is a duplicate of `AsnMixin_Image`. One should be able to just use `AsnMixin_image` at this point.

Reply:
fixed in latest commit